lists_and_tuples/task_04.py

```diff
@@ -0,0 +1,7 @@
+week_days = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+
+print(f"First day: {week_days[0]}")
+print(f"Last day: {week_days[-1]}")
+print(f"Middle days: {week_days[1:6]}")
```

Comment:
It is better to use `-1` in case we do not know what is length of the sequence
This way we ensure that we will get middle values without knowing the length of the list

```suggestion
print(f"Middle days: {week_days[1:-1]}")
```